endpoints/openrtb2/auction.go

```diff
@@ -1334,6 +1339,44 @@ func validateDevice(device *openrtb2.Device) error {
 	return nil
 }
 
+func validateOrFillChannel(reqWrapper *openrtb_ext.RequestWrapper, isAmp bool) error {
+	requestExt, err := reqWrapper.GetRequestExt()
+	if err != nil {
+		return err
+	}
+	requestPrebid := requestExt.GetPrebid()
+
+	if requestPrebid == nil || requestExt.GetPrebid().Channel == nil {
```

Comment:
**Nitpick:** You can replace `requestExt.GetPrebid()` in the second part of this if condition to just `requestPrebid` as you've already saved that in the `requestPrebid` variable.